core/controller/src/main/scala/org/apache/openwhisk/core/controller/actions/PrimitiveActions.scala

```diff
@@ -569,7 +569,9 @@ protected[actions] trait PrimitiveActions {
       }
     }
 
-    activationStore.store(activation, context)(transid, notifier = None)
+    if (user.limits.storeActivations.getOrElse(true)) {
```

Comment:
Can/should we move this into the ActivationStore itself? The user's identity is available in there as well.

I guess the thought was to not do that, because then each store needs to make sure to implement this if? Can we instead maybe come up with a wrapper function for all `activationStore.store` calls that takes care of a general concern like this?

Reply:
Now, I introduced the method `storeAfterCheck`, to do exactly that in ActivationStore, independant of the implementation.